09-facebook.Rmd

```diff
@@ -3,52 +3,66 @@
+✔️ can content be shared privately?  
+Yes, there are multiple methods of sharing content privately with someone else on Facebook. The simplest way is by messaging them directly via Messenger and attaching files. Another method for private sharing is by creating a post on one’s personal feed and customizing the audience for the post so that only specific friend(s) can view the post. Sharing content posted by other users can also be done privately by sending them in Messenger and sharing to the personal feed privately.    
+✔️ how is content shared?  
+The main method for sharing content is by directly creating a post, which can be customized to be shown to different audiences. Users can photos, videos, and gifs along with other additional content when creating a post. It’s also possible host Q&As, stream live video, tag other people, which provide engagement and dynamic options for users to interact. Moreover, there is an option for people to create private photo or text stories that will be visible for 24 hours on Facebook and Messenger. If users want to share content that other people have created, they can simply go to the post and tap the share button. They can choose one of the following options:  
+•	Share Now (Friends)  
+•	Share to Feed  
+•	Share to Your Story (Friends)  
+•	Send in Messenger  
+•	Share on a friend’s profile  
+✔️ how does content go viral?  
+One of the many ways to go viral is to post something that 	is related to the news, trending topics, or other major events that are happening in the real world. There will be many users who are searching and viewing posts about trending topics so there is a higher chance for a post to go viral if it’s socially relevant. In addition, most viral content is short and original. Users have a very short attention span and are often quickly scrolling through their timeline, so if a post catches someone’s attention, it’s very likely that it’s short and doesn’t copy other content.   
+✔️ is the parent company public or private?  
+Facebook first went public in 2012 as the shares became available for selling in public markets. Note that Mark Zuckerberg announced on Oct. 28th, 2021, that the name of the parent company (Facebook Inc.) is now Meta Platforms Inc.   
+✔️ who are the major investors?  
+According to Investopedia, the major insider shareholders are Michael Schroepfer, David Fischer, and David Wehner. Also, the major institutional shareholders of the company are Mark Zuckerberg, Vanguard Group Inc., and BlackRock Inc.   
+✔️ how does the company generate revenue?  
+The company generate most of its revenue by showing digital advertisements. Facebook’s business model is explained simply by Mark Zuckerberg in the video below.   
+https://www.youtube.com/watch?v=n2H8wx1aBiQ  
+References:  
```

Comment:
Please create references as footnotes. Instructions are on the Major Project page.

Reply:
@cmadland Thanks for the reminder @cmadland. I'll add footnotes for the next pull request.